data-lineage/src/main/java/com/wipro/ats/bdre/lineage/LineageMain.java

```diff
@@ -638,7 +653,7 @@ private void generateLineageDot() {
 
 		StringBuilder middle = new StringBuilder();         // all nodes definitions
 		// input tables to override output tables if duplicates are found
-		for (Node node : finalOutTableNodes) {
+		/*for (Node node : finalOutTableNodes) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3ACommentedOutCodeLine)